src/server_manager/web_app/app.ts

```diff
@@ -88,6 +85,13 @@ async function computeDefaultAccessKeyDataLimit(
   }
 }
 
+// Returns whether the user has seen a disclaimer for the updated feature metrics data collection
+// policy.
+function hasFeatureMetricsConsent(): boolean {
```

Comment:
I'm sorry, I misunderstood on first read. So it's not really consent, because we can't collect the metric unless the metrics sharing is enabled. It's really about notification.
So maybe call it something like `notifiedOfFeatureMetrics()`

Reply:
Done.